src/components/Events/GM/GM.js

```diff
@@ -130,40 +95,38 @@ function GMLiveProgram() {
 					<h2>Program</h2>
 					<div className="row">
-						<p className="timestamps">6:35 p.m. PST</p>
+						<p className="timestamps"> {gmData.times.welcome + ' ' + gmData.times.timezone} </p>
 						<div className="row-description-wrapper">
-							<h3 className="row-description-heading">Rucha Patki: Welcome</h3>
+							<h3 className="row-description-heading">{gmData.welcomeSpeaker}: Welcome</h3>
 						</div>
 					</div>
 					<div className="row">
-						<p className="timestamps">6:40 p.m. PST</p>
+						<p className="timestamps">{gmData.times.committees + ' ' + gmData.times.timezone}</p>
 						<div className="row-description-wrapper">
 							<h3 className="row-description-heading">Committee Presentations</h3>
-							<p className="row-description-subheading">Learn what each of ACM&rsquo;s eight committees has in store for winter quarter</p>
+							<p className="row-description-subheading">Learn what each of ACM&rsquo;s eight committees has in store for {gmData.quarter} quarter</p>
 							{renderCommitteeSpeakers()}
 						</div>
 					</div>
 					<div className="row">
-						<p className="timestamps">7:05 p.m. PST</p>
+						<p className="timestamps">{gmData.times.board + ' ' + gmData.times.timezone}</p>
 						<div className="row-description-wrapper">
 							<h3 className="row-description-heading">ACM Board</h3>
-							<p className="row-description-subheading">See what ACM wide event ACM Board has planned.</p>
-							<p><span className="board"><img src={process.env.PUBLIC_URL + '/images/logo.png'} alt="Logo for ACM" className="committee-logos"/>CS Town Hall</span>: Katherine Chong</p>
-							<p><span className="board"><img src={process.env.PUBLIC_URL + '/images/logo.png'} alt="Logo for ACM" className="committee-logos"/>Moonshots</span>: Sharvani Jha</p>
-							<p><span className="board"><img src={process.env.PUBLIC_URL + '/images/logo.png'} alt="Logo for ACM" className="committee-logos"/>Tech Gala</span>: Daniel Jaffe, Evan Zhong</p>
+							<p className="row-description-subheading">See what ACM wide events ACM Board has planned.</p>
+							{renderBoardEvents()}
 						</div>
 					</div>
 					<div className="row">
-						<p className="timestamps">7:10 p.m. PST</p>
+						<p className="timestamps">{gmData.times.social + ' ' + gmData.times.timezone}</p>
 						<div className="row-description-wrapper">
 							<h3 className="row-description-heading">Tabling and Social</h3>
-							<p className="row-description-subheading">Join us in our official <a href="https://bit.ly/ACMdiscord"  target="_blank" rel="noopener noreferrer" id="acm-discord-invite-cta">ACM Discord</a> for Committee tabling and socials</p>
-							<p>Daniel Jaffe, Evan Zhong</p>
+							<p className="row-description-subheading">Join us in our official <a href={gmData.links.discord}  target="_blank" rel="noopener noreferrer" id="acm-discord-invite-cta">ACM Discord</a> for Committee tabling and socials</p>
+							{/* <p>Daniel Jaffe, Evan Zhong</p> */}
```

Comment:
I would actually remove this line entirely, or generalize it rather than commenting it out